helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -232,7 +232,7 @@ spec:
         {{- else }}
         - /otelcol
         - --config=/conf/relay.yaml
-        {{- if eq .Values.distribution "gke/autopilot" }}
+        {{- if and (eq .Values.distribution "gke/autopilot") (eq .Values.logsEngine "fluentd") }}
         {{- /* Temporary no-op argument required to run collector on GKE Autopilot */}}
         - --metrics-addr=0.0.0.0:8889
```

Comment:
No related to this PR, but out of curiosity, could this be localhost? I assume this is the listener addr for prometheus endpoint and will be scraped by the locally running collector.

Reply:
This is a noop. The collector ignores this argument. It's just need to bypass the allowlist

Reply:
But your Q still holds true. It's just provided in the config now https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl#L646. I think you're right it should be localhost. We should discuss it more.